src/components/Counter.js

```diff
@@ -1,39 +1,40 @@
 import React, { Component } from "react";
 import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { increment, decrement, incrementIfOdd, incrementAsync  } from '../actions';
```

Comment:
You actually don't need to create new actions to implement the stretch functionality, but if you can that's awesome!